VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/FeatureTest.java

```diff
@@ -10,11 +10,13 @@
 
 @RunWith(AndroidJUnit4.class)
 public abstract class FeatureTest {
+    EdxEnvironment environment;
```

Comment:
This should be made `protected` I guess, to limit it's access to only subclasses, since they're the only valid accessors of it.